### PR TITLE
Copy data files as part of build

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Review by matthew-brett — COMMENTED** [review]
Shouldn't we be copying the data files from `notebooks`?   The notebook build copies only the data for the notebooks to that directory.  Any way of fetching the `notebooks` directory from the `nb-dir` config?

File: source/generate-ninja.py [hunk 1]
```diff
@@ -46,6 +47,8 @@ def replace_ext(fn, new_ext):
     for (diagram, built_diagram) in zip(diagrams, built_diagrams[fmt]):
         w.build(built_diagram, 'svg2x', diagram)
 
+data_files = glob('data/*')
```
[the remark this answers is not in the document]
Comment (stefanv) [reply source/generate-ninja.py:50]: Yes, let me also modify the line below.

[the remark this answers is not in the document]
Comment (stefanv) [reply source/generate-ninja.py:50]: Hrm, wait, so targets depend on built output? I think that's going to be harder to do.

[the remark this answers is not in the document]
Comment (stefanv) [reply source/generate-ninja.py:50]: How do we know, at the point where we run `generate-ninja.py`, which data files are implicated?

[the remark this answers is not in the document]
Comment (stefanv) [reply source/generate-ninja.py]: For now, copying all data files into configurable notebook dir.

**Comment by matthew-brett** [comment]
Thanks!

**Pull request merged**